gemfirexd/core/src/main/java/com/pivotal/gemfirexd/internal/impl/sql/GenericStatement.java

```diff
@@ -653,7 +653,8 @@ private final PreparedStatement prepMinion(
 				  StringBuilder queryTextForStats = null;
 				  boolean continueLoop = true;
 			    int i = 0;
-			    boolean forceSkipQueryInfoCreation = false;
+          boolean forceSkipQueryInfoCreation = Misc.getMemStore().isSnappyStore()
+            && lcc.getBucketIdsForLocalExecution() != null;
```

Comment:
Formatting looks out of place. Perhaps can copy the indentation of previous line and copy.